velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -17,6 +17,8 @@
 #include "velox/common/base/tests/GTestUtils.h"
 #include "velox/dwio/parquet/tests/ParquetTpchTestBase.h"
 
+using namespace facebook::velox::exec::test;
```

Comment:
Is this needed?

Reply:
Yes, `ParquetTpchTestBase` and `assertQuery` used by `ParquetTpchTest` are in this namespace.